test/functional/wallet_mnemonicbits.py

```
@@ -17,14 +17,64 @@ def set_test_params(self):
     def skip_test_if_missing_module(self):
         self.skip_if_no_wallet()
 
+    def get_mnemonic(self, node):
```

Ruff (0.12.2)

Missing return type annotation for public function `get_mnemonic`

(ANN201)